cli/prisma2/src/bin.ts

```diff
@@ -99,18 +93,18 @@ async function main(): Promise<number> {
     console.error(result)
     return 1
   }
-
+  // check prisma for updates
   const checkResult = await checkpoint.check({
     product: 'prisma',
     version: packageJson.version,
-    cache_only: true,
     disable: isCI,
   })
-  if (checkResult && checkResult.outdated) {
+  // if the result is cached and we're outdated, show this prompte
+  if (checkResult.status === 'ok' && checkResult.data.outdated) {
     console.error(
-      `\n${chalk.blue('Update available')} ${packageJson.version} -> ${checkResult.current_version}\nRun ${chalk.bold(
-        'npm install -g prisma2',
-      )} to update`,
+      `\n${chalk.blue('Update available')} ${packageJson.version} -> ${
+        checkResult.data.current_version
+      }\nRun ${chalk.bold(checkResult.data.current_download_url)} to update`,
```

Comment:
strange naming

Reply:
yah, it's meant to be API compatible to allow us to use other checkpoint clients.

Reply:
New property `install_command` would probably be the easier choice. (But see [my other comment](https://github.com/prisma/prisma2/pull/1197/files#r362086962) as well)

Reply:
sure, i'll change this.